test/system/learning_status_test.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require 'application_system_test_case'
+
+class LearningStatusTest < ApplicationSystemTestCase
+  test 'learning status changes to submitted after the mentor cancels the confirmation' do
+    visit_with_auth "/products/#{products(:product8).id}", 'machida'
+    click_button '担当する'
+    click_button '提出物を確認'
+    click_button '提出物の確認を取り消す'
+    visit_with_auth course_practices_path(courses(:course1).id), 'kimura'
+    assert_no_selector 'a', text: '修了'
```

Comment:
`click_button '提出物の確認を取り消す'` 部分をコメントアウトしてもテストが通っているようです。

コースの一覧よりも提出物ページに直接アクセスし、『修了』文言がないことを確認するのはどうでしょうか？

```suggestion
    visit_with_auth "/products/#{products(:product8).id}", 'kimura'
    assert_no_selector 'h2', text: '修了'
```

Reply:
ご指摘感謝します。
textのみ変更して修正しました。

### 余談
そもそもこの修了の `a`タグが `div` の内部にあったので、正しくテストできていませんでした。
(`assert_no_selector`が希望の動作をしていなかった。）